superset-frontend/packages/superset-ui-core/src/number-format/factories/createDurationFormatter.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import prettyMsFormatter from 'pretty-ms';
+import prettyMilliseconds, { Options } from 'pretty-ms';
```

Comment:
Renaming the default export to `prettyMilliseconds`, as that's what it's called in the source, and pull in `Options` via the newly exposed named export.